src/test/scala/temple/generate/orchestration/dockercompose/DockerComposeGeneratorTest.scala

```diff
@@ -7,8 +7,15 @@ class DockerComposeGeneratorTest extends FlatSpec with Matchers {
 
   behavior of "DockerComposeGenerator"
 
-  it should "generate an empty file" in {
-    val generated = DockerComposeGenerator.generate("example", UnitTestData.basicOrchestrationRootWithMetrics)
-    generated.head._2 shouldBe ""
+  it should "generate a docker-compose file for a service without metrics" in {
+    val (_, fileContents) =
```

Comment:
Just curious, but could this be `_ -> fileContents` instead?

Reply:
It could indeed. It might look a little odd, but so does so much of Scala